lightning/src/routing/scoring.rs

```diff
@@ -650,6 +650,20 @@ struct HistoricalBucketRangeTracker {
 
 impl HistoricalBucketRangeTracker {
 	fn new() -> Self { Self { buckets: [0; 8] } }
+
+	/// Returns the average of the buckets between the two trackers.
+	pub(crate) fn combine(mut self, other: Self) -> Self {
+		let mut buckets: [u16; 8] = [0; 8];
+		for i in 0..8 {
```

Comment:
Let's either make a constant for the length or `zip` two iterators. I think there's a PR to change the number of buckets, so we don't want to inadvertently miss updating this.